src/DataStructs.h

```diff
@@ -90,6 +90,7 @@ struct SimParams {
 	string tracking = "None";		// meltpool tracking mode
 	double dt = 1e-5;			// timestep
 	int out_freq = 1;		// output frequency
+	double radiusCheck = 1.0;		// The radius to be checking perimeter points from (different behavior is >1)
```

Comment:
Typo in comment? `different` -> `default`?

Reply:
The whole "()" probably shouldn't be there at all. I plan on making the behavior different when >1 to make it more optimal but that is a TODO